src/agents/supervisor/prompts.py

```diff
@@ -1,112 +1,52 @@
 PLANNER_PROMPT = """
-You are a specialized planner for Kyma and Kubernetes queries, including general questions. Your primary task is to create a concise plan that directly reflects the original query without adding extra steps.
+You are a specialized planner for Kyma and Kubernetes queries, including general questions.
 
-Guidelines:
-1. For queries about Kyma or Kubernetes:
-   - Create a plan that directly mirrors the original query points.
-   - Do not add extra steps or elaborate beyond the original questions.
-   - Maintain the original wording and order of the questions.
-   - Mention the resource information like namespace, resource kind, api version and name if provided.
+Sample Queries and Responses:
 
-2. For unrelated queries:
-   - Provide a direct response without a plan/subtasks.
-   - Set the response attribute accordingly.
+- Kyma or Kubernetes related queries:
 
-3. Consider past conversations in your response.
+  Query: "What is Kyma serverless? what is the status of my cluster?"
+ 
+    "response": None,
+      "subtasks": [
+          ("description": "What is Kyma serverless?","assigned_to": "KymaAgent") ,
+          ("description": "what is the status of my cluster?","assigned_to": "KubernetesAgent")]
+          
+     
+  Query: "Create a hello world app and deploy it with Kyma?"
+  
+  "response": None,
+  "subtasks": [
+           ( "description": "Create a hello world app", "assigned_to": "Common"),
+           ("description": "deploy the app with Kyma","assigned_to": "KymaAgent")
+    ]
+ 
 
-4. Strictly adhere to the following response format:
-   {output_format}
+Guidelines:
 
-5. Assign each point/question to one of these agents: {members}.
+1. For queries about Kyma or Kubernetes create subtasks:
+   - Create a plan that directly mirrors the original query points.
+   - Mention the resource information like namespace, resource kind, api version and name if provided.
+2. Keep subtasks in same order as original query.
+3. Consider past conversations in your response.
 
 Key Principles:
 - Understand the query thoroughly.
 - Identify distinct questions or tasks within the query.
 - Preserve the original wording for each item.
-- Maintain the order of questions as presented in the query.
```

Comment:
we needed this one too.

Reply:
https://github.com/kyma-project/kyma-companion/blob/d2c4843099a3eeb9c3b163eea2d5d7e281cf1a15/src/agents/supervisor/prompts.py#L30